indexer-app/src/cmr/indexer/data/index_set.clj

```diff
@@ -98,6 +98,15 @@
    :originator-id.lowercase m/string-field-mapping
    :tag-value.lowercase m/string-field-mapping})
 
+(defnestedmapping variables-mapping
+  "Defines mappings for variables."
+  {:measurement m/string-field-mapping
+   :measurement.lowercase m/string-field-mapping
+   :variable m/string-field-mapping
+   :variable.lowercase m/string-field-mapping
+   :originator-id.lowercase m/string-field-mapping
+   :variable-value.lowercase m/string-field-mapping})
```

Comment:
That is for future. For tag association, we can provide a string type value when associate a tag to a collection which can be searched by later. I am doing the same thing here for variable association, similar to `originator-id` here even though we don't have requirement for it either. We can take it off if you think we won't have string value association during variable association.